sql/src/main/java/io/crate/exceptions/PermissionDeniedException.java

```diff
@@ -0,0 +1,21 @@
+package io.crate.exceptions;
+
+
+import io.crate.operation.user.User;
+
+import java.util.Locale;
+
+public class PermissionDeniedException extends ResourceUnknownException {
```

Comment:
@andreidan i wasn't sure which exception should be extended .. what do you suggest ?